cartographer_ros/cartographer_ros/map_builder_bridge.h

```diff
@@ -25,10 +25,11 @@
 #include "cartographer_ros/node_options.h"
 #include "cartographer_ros/sensor_bridge.h"
 #include "cartographer_ros/tf_bridge.h"
-#include "cartographer_ros_msgs/SubmapEntry.h"
-#include "cartographer_ros_msgs/SubmapList.h"
-#include "cartographer_ros_msgs/SubmapQuery.h"
-#include "nav_msgs/OccupancyGrid.h"
+#include "cartographer_ros_msgs/msg/submap_entry.hpp"
+#include "cartographer_ros_msgs/msg/submap_list.hpp"
+#include "cartographer_ros_msgs/srv/submap_query.hpp"
+#include "nav_msgs/msg/occupancy_grid.hpp"
+#include <rclcpp/rclcpp.hpp>
```

Comment:
I mostly just kept the quote vs bracket distinction as they were in the original cartographer code.  That being said, I think we prefer brackets for ROS2 headers, so I'll change those (this won't make any difference for merging from upstream, since I'm changing those lines anyway).